econplayground/main/migrations/0029_auto_20180312_1518.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 2.0.3 on 2018-03-12 19:18
+
+from decimal import Decimal
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0028_auto_20180228_1503'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='jxglinetransformation',
+            old_name='x',
+            new_name='x1',
+        ),
+        migrations.RenameField(
+            model_name='jxglinetransformation',
```

Comment:
Just verifying: the old y -> x2 and the old z -> x3?

Reply:
Actually, it looks like that is a mistake. But, it really doesn't matter since the existing model isn't in use yet.

Reply:
np - I will merge.